.mergify.yml

```diff
@@ -1,11 +1,36 @@
+---
+
+queue_rules:
+  - name: default
+    conditions:
+      - '#approved-reviews-by>=2'
+      # Conditions to get out of the queue (= merged)
+      # Following to be uncommented once Github CI is set up
+      #- check-success=flake8
+      #- check-success=integration
+      #- check-success=test (2.7)
+      #- check-success=test (3.6)
+      #- check-success=test (3.7)
+      #- check-success=test (3.8)
+      #- check-success=test (3.9)
 pull_request_rules:
-  - name: automatic merge when CI passes and approvals
+  - name: automatic merge
     conditions:
-      - "#approved-reviews-by>=1"
-      - status-success=continuous-integration/travis-ci/pr
-      - base=master
+      - label!=(no-mergify|WIP)
+      - '#approved-reviews-by>=2'
+      # We have to duplicate these because mergify won't allow us to use an
+      # anchor
+      # Following to be uncommented once Github CI is set up
+      #- check-success=flake8
+      #- check-success=integration
+      #- check-success=test (2.7)
+      #- check-success=test (3.6)
+      #- check-success=test (3.7)
+      #- check-success=test (3.8)
+      #- check-success=test (3.9)
```

Comment:
Since we are updating anyway, do we want to include 3.10 in both of these? We should do that across the rest of the projects as well, but seems like a good time for this one, since the changes are underway already.